src/app/utils/url.ts

```diff
@@ -1,4 +1,4 @@
-const validProtocols = ['http:', 'https:', 'ftp:', 'ipfs:', 'data:']
+const validProtocols = ['http:', 'https:', 'ftp:', 'ipfs:', 'data:', 'mailto:']
```

Comment:
This is dangerous https://github.com/oasisprotocol/oasis-wallet-web/issues/228

Reply:
Right, but to frame this properly,  currently the only place we are using mailto links are the faucet links, which are hardwired in the code.

Maybe we should add this as an exception for the faucet link only, and not in the general frunction?

Reply:
> Right, but to frame this properly, currently the only place we are using mailto links are the faucet links, which are hardwired in the code.

no, it was ensuring safety of validator logos and NFT links too

Reply:
> Maybe we should add this as an exception for the faucet link only, and not in the general frunction?

sure, we can consider all our external links as safe